airflow/utils/db.py

```diff
@@ -930,6 +930,36 @@ def check_conn_id_duplicates(session: Session) -> Iterable[str]:
         )
 
 
+def check_username_duplicates(session: Session) -> Iterable[str]:
+    """
+    Check unique username in User & RegisterUser table
+
+    :param session:  session of the sqlalchemy
+    :rtype: str
+    """
+    from airflow.www.fab_security.sqla.models import RegisterUser, User
+
+    for model in [User, RegisterUser]:
+        dups = []
+        try:
+            dups = (
+                session.query(model.username)  # type: ignore[attr-defined]
+                .group_by(model.username)  # type: ignore[attr-defined]
+                .having(func.count() > 1)
+                .all()
+            )
+        except (exc.OperationalError, exc.ProgrammingError):
+            # fallback if tables hasn't been created yet
+            session.rollback()
+        if dups:
+            yield (
+                f'Seems you have non unique username in {model.__table__.name} table.\n'  # type: ignore
+                'You have to manage those duplicate usernames '
```

Comment:
This needs to say something about mixed-case, as otherwise users could be forgiven for thinking that `Test` and `test` are different users.

And "manage" isn't clear either -> "delete or rename"

Reply:
Addressed. You can take another look. Thanks